pyproject.toml

```diff
@@ -1,3 +1,41 @@
 [tool.black]
 line-length = 100
-target-version = ['py39']
\ No newline at end of file
+target-version = ['py39']
+
+[tool.poetry]
+name = "trackthenews"
+version = "0.2"
+description = "Monitor RSS feeds for keywords and act on matching results. A special project of the Freedom of the Press Foundation."
+authors = ["Freedom of the Press Foundation <info@freedom.press>"]
+maintainers = [
+    "Harris Lapiroff <harris@freedom.press>",
+    "Erik Moeller <erik@freedom.press>",
+]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/freedomofpress/trackthenews"
+include = ["trackthenews/fonts/*"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+feedparser = "^6.0"
+future = "^0.17.1"
+html2text = "2018.1.9"
+Pillow = "^8.4"
+pyaml = "^18.11.0"
+readability-lxml = "^0.8.1"
+requests = "^2.31.0"
+tweepy = "^4.14.0"
+"Mastodon.py" = "^1.8.1"
+
+[tool.poetry.group.dev.dependencies]
+black = "^23.7.0"
+flake8 = "^6.0.0"
+isort = "^5.12.0"
+
+[tool.poetry.scripts]
+trackthenews = 'trackthenews:main'
+
```

Comment:
IMO we ought to apply some basic configuration to isort here as well, e.g.

```
[tool.isort]
profile = "black"
multi_line_output = 3
```

See documentation under [Compatibility with Black](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html) and [Multi-line output modes](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html) (no intent to start a war here but I think everything except 3 is kinda not good).

Reply:
I would tend to agree. Either 3 or `force_single_line`. @eloquence @SaptakS any opinions?

Reply:
We only set `profile="black"` on our other projects AFAICT, but no opinion either way.